packages/hardhat/contracts/GardensTemplate.sol

```diff
@@ -19,72 +26,123 @@ contract GardensTemplate is BaseTemplate, AppIdsXDai {
     uint256 private constant TOKEN_MAX_PER_ACCOUNT = uint256(-1);
     address private constant ANY_ENTITY = address(-1);
     uint8 private constant ORACLE_PARAM_ID = 203;
+    uint256 public constant AVERAGE_PRICE_PERIOD = 86400; // 24 hours
+    uint8 public constant UPDATE_FREQUENCY = 8; // 8 times within price period
+    uint256 public constant UPDATE_PERCENT_REWARD = 2e16; // 2% reward per update call
+    address private constant BURN_ADDRESS = 0x000000000000000000000000000000000000dEaD;
+    uint256 public constant XDAI_IN_HNY_REQUIRED_FOR_NEW_TOKEN = 100e18;
```

Comment:
This amount seems on the lower side in my opinion. Wonder what you think of make it configurable so the summoner of a Garden can decide as well and we don't have to deploy a new template version if we decide in the future that a different value is a better fit.

Reply:
This requires discussion in the gardens chat.